src/main/java/io/github/tryexceptelse/jdex/fe/Gui.java

```diff
@@ -1,10 +1,21 @@
 package io.github.tryexceptelse.jdex.fe;
 
+import io.github.tryexceptelse.jdex.JDex;
+
 /**
  * Graphic User Interface. extends Ui
  */
 public class Gui implements Ui {
```

Comment:
why Ui not IUi? In the rest of our classes we use IClass for the interface and Class for the class.